pkgs/by-name/va/vanguards/package.nix

```
@@ -0,0 +1,48 @@
+{ pkgs, lib }:
+pkgs.python312Packages.buildPythonPackage rec {
```

```suggestion
python312Packages.buildPythonApplication rec {
```
This is only an application and not a library, right?